client/sync.js

```diff
@@ -75,45 +75,49 @@ const maybeSetDeviceId = (requester) => {
  * @param {RequestUtil} requester
  */
 const startSync = (requester) => {
+  /**
+   * Helper method to convert s3objects to decrypted JS records
+   * @param {Array.<Uint8Array>} s3Objects
+   * @returns  {Array.<Object}
+   */
+  const getJSRecords_ = (s3Objects) => {
```

Comment:
it's a JS convention for a private method/var, same as the prefix usage of underscore